### ISSUE TITLE
What we are calling users on the cap table is inconsistent. Are they Shareholders or Option holders?

### ISSUE THREAD

**Issue by alisoniven** [sharewave/frontus#1621]
What we are calling users on the cap table is inconsistent. Are they Shareholders or Option holders?
![2](https://cloud.githubusercontent.com/assets/6333864/6022749/16156474-ab8d-11e4-9a02-c85f19f18cb2.png)
![1](https://cloud.githubusercontent.com/assets/6333864/6022748/16134f0e-ab8d-11e4-9d26-d3ad3f91eed1.png)

**Comment by BertHartm** [comment]
Should we drop that sub heading altogether? It's not particularly useful in
this context I think (this context being anywhere we're looking at a
transaction).

**Comment by alisoniven** [comment]
i would agree with that. but we would need to re-organize the rail a bit (especially when details/docs tabs are there). Do you guys want to start customizing these rails for their context?

**Comment by RWalport** [comment]
The layout might be wrong but the option holder vs shareholder is correct. There are 2 transactions here and if we're going to label the owner it needs to be as it is, the grant has an option holder, the purchase has a shareholder (this contact is both). Maybe we don't need to list the name at all here, but on the ledger/cap table we really should be using these labels.

**Comment by alisoniven** [comment]
It just seemed strange in context. I'll try to sort something out with the details on rail removing the name.
